.circleci/config.yml

```diff
@@ -1106,8 +1089,14 @@ jobs:
       - run:
           name: Run fastlane swiftlint lane
           command: |
-            bundle exec fastlane run swiftlint raise_if_swiftlint_error:true strict:true \
-            reporter:junit output_file:fastlane/test_output/swiftlint/junit.xml
+            # FIXME This doesn't work because somehow swift package plugins run for every target/module/scheme?
+            #  Only the last of those runs will save the output file, causing issues to be lost.
+            swift package plugin \
+              --allow-writing-to-directory fastlane/test_output/swiftlint \
+              swiftlint \
+              --strict \
+              --reporter junit \
+              --output fastlane/test_output/swiftlint/junit.xml
```

Comment:
This doesn't work because somehow swift package plugins run for every target/module/scheme? Only the last of those runs will save the output file, causing issues to be lost.

Reply:
Running it with verbose output tells me it's running for each module, which map to the targets specified in our `Package.swift`: 
```shell
info: Finished running in module 'RevenueCat_CustomEntitlementComputation'
info: Finished running in module 'RevenueCatUITests'
info: Finished running in module 'RevenueCatUI'
info: Finished running in module 'RevenueCat'
info: Finished running in module 'ReceiptParserTests'
info: Finished running in module 'ReceiptParser'
```

I think this is `swift package` that's running the `plugin` command for each target. It seems to be happening outside of `swiftlint`'s control. Not sure yet how to make `swift package` confine itself to a single target.

Reply:
There's a [convention](https://github.com/swiftlang/swift-package-manager/blob/main/Documentation/Plugins.md#implementing-the-command-plugin-script) to support a `--target` argument.  SwiftLint's command plugin supports it partly. It passes the `--target` argument on to the actual `swiftlint` program, which doesn't recognize it and errors out. 

There’s [a fork](https://github.com/dshikulin-mwb/SwiftLintPlugins) that fixes this issue. I’ve [asked](https://github.com/SimplyDanny/SwiftLintPlugins/pull/1#issuecomment-2346144183) if they can propose their fix upstream.

Reply:
0.57.1 includes a fix for this.